sources/packages/backend/workflow/src/workflow-definitions/assessment-gateway.bpmn

```diff
@@ -923,6 +919,48 @@ Note: the workflow process should continue its execution as a regular reassessme
     <bpmn:sequenceFlow id="change-in-progress-no-pir-required-flow" name="Change is in progress" sourceRef="pir-change-request-gateway" targetRef="Gateway_04uphc0">
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">=applicationEditStatus = "Change in progress"</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_0jltxxn" sourceRef="change-request-workflow-wrap-up-task" targetRef="minitry-approval-change-declined-end-event" />
+    <bpmn:serviceTask id="change-request-workflow-wrap-up-task" name="Workflow wrap up (Assessment status only)">
+      <bpmn:extensionElements>
+        <zeebe:taskDefinition type="workflow-wrap-up" />
+        <zeebe:taskHeaders>
+          <zeebe:header key="wrapUpType" value="Assessment status only" />
```

Comment:
It could be a variable initialized at start of workflow and used across many places. 

![image](https://github.com/user-attachments/assets/4e3c2ea3-4302-4087-8b36-30c7921827c5)

Reply:
But what is mentioned above is not about usage of headers. It is about using a value from variable instead of repeating the value. Let me know if I am missing something here.

Reply:
I got it now that it is related to a broader conversation about the enum usage in the workflow and a possible way to have an enum-like structure type in BPMN. I will take a look outside this PR effort. Thanks for the chat.